Framework/HistogramData/src/HistogramRebin.cpp

```diff
@@ -0,0 +1,157 @@
+#include "MantidHistogramData/HistogramRebin.h"
```

Comment:
I would rename this (and the other related files) to `Rebin.h`. Since it is in the `HistogramData` module it will be obvious, people can just use:

```cpp
#include "MantidHistogramData/Rebin.h"
```